R/input-formats.R

```diff
@@ -3,24 +3,57 @@
 #' In order to format validation docs, mrgvalidate must collect various pieces
 #' of information from the caller. This information can be broken down into two
 #' categories: information about the requirements and information about the
-#' executed tests. Below is a description of how information about the executed
-#' tests is expected to be formatted.
+#' executed tests. Below is a description of how this data is expected to be
+#' formatted. **Note: there a companion package `mrgvalprep` containing
+#' functions to preprocess common inputs (R's `testthat`, Googlesheets, etc.)
+#' into the format required by `mrgvalidate`.
 #'
 #' @details
 #'
+#' ## Stories and requirements
+#'
+#' A tibble or data.frame containing the stories, and optionally requirements,
+#' for the software being validated. Will be passed to `specs` argument of
+#' [create_validation_docs()]. Must have the following columns:
+#'
+#' * StoryId: unique identifier for the story
+#'
+#' * StoryName: short name of story (ideally 5-8 words max, but no hard limit)
+#'
+#' * StoryDescription: body of the story ("As a user,...")
+#'
+#' * ProductRisk: specify the risk to the product of this story (typically
+#' "Low", "Medium", "High", etc.)
+#'
+#' * RequirementId (Optional): unique identifier of technical requirement
+#' associated with story. If included, there can be >1 requirement per story,
+#' each on its own line, in which case the first four columns will be repeated
```

Comment:
I tripped over "on its own line" at first.  Given the context, it might be clear enough that "on its own line" isn't referring to within the cell (e.g., in the same way test IDs could be newline-separated), but perhaps something like "each should have its own row in the table" would avoid potential confusion.